Yolov8 notebooks update ignored scope

kshpv opened the pull request:
Update ignored scope at NNCF optimization step utilizing subgraph approach.
This fixes some extra quantizers and add more readability.

Commit:
update ignored scope

kshpv commented:
I also removed the output of all cells. If it is not expected, please let me know I will rollback

Commit:
add comment

Commit:
Merge remote-tracking branch 'remote/latest' into yolov8_ignored_scope_upd

eaidova commented:
> I also removed the output of all cells. If it is not expected, please let me know I will rollback

@kshpv thanks, yes it is not expected to have removed outputs, please return back

kshpv commented:
> > I also removed the output of all cells. If it is not expected, please let me know I will rollback
> 
> @kshpv thanks, yes it is not expected to have removed outputs, please return back

updated

Commit:
add outputs
